shared/src/main/java/io/crate/exceptions/Exceptions.java

```diff
@@ -36,6 +36,14 @@ public static void rethrowUnchecked(final Throwable ex) {
         Exceptions.rethrow(ex);
     }
 
+    public static <T> T rethrowRuntimeException(Throwable t) {
```

Comment:
It took me some time to understand what the purpose of `T` is here.  The usage in the TableCreator makes it clear.